kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md

```diff
@@ -1,14 +1,17 @@
 # WebLogic sample domain home in Docker image
 
-The sample scripts demonstrate the creation of a WebLogic domain home in a Docker image. The scripts also generate the domain YAML file, which can then be used to start the Kubernetes artifacts of the corresponding domain. Optionally, the scripts start up the domain, and WebLogic Server pods and services.
+The sample scripts demonstrate the creation of a WebLogic domain home in a Docker image. The log home is on an existing Kubernetes persistent volume (PV) and persistent volume claim (PVC). The scripts also generate the domain YAML file, which can then be used to start the Kubernetes artifacts of the corresponding domain. Optionally, the scripts start up the domain, and WebLogic Server pods and services.
```

Comment:
Should this sentence be qualified regarding whether logHomeOnPV is enabled?
"The log home is on an existing Kubernetes persistent volume (PV) and persistent volume claim (PVC)."

For example, "If `logHomeOnPV` is enabled, then the log home resides on an existing Kubernetes persistent volume (PV) and persistent volume claim (PVC)."

Reply:
Fixed in the new change. Thanks